bridge/server/utils/logger.ts

```diff
@@ -35,13 +39,19 @@ export class LoggerImpl {
     if (this._log == null) {
       return;
     }
-    // Print debug levels only if the component is enabled
-    if (level === Level.DEBUG && this._components != null && this._components[component] !== true) {
+
+    const componentConfigured = this._components != null && this._components[component] === true;
```

Comment:
I just realized we have wrong checks here. `this._log` and `this._components` can't even be `null` but `undefined`